ebusd/CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 <!-- https://developers.home-assistant.io/docs/add-ons/presentation#keeping-a-changelog -->
 
+## TBC
+
+- BREAKING: Remove old style loglevel and logareas option
+- BREAKING: Remove TCP port option.  Port will default to port 8888 internally.  External port can be configured in network options
+
+- IMPROVE: Add access level config option
+- IMPROVE: Enable custom config files (save in share folder)
+- IMPROVE: Configure logger for individual areas
+- IMPROVE: Allow wireless ebusd adapter
```

Comment:
I think we need to try and streamline this a bit more.  Maybe change IMPROVE to BETA??

Reply:
Changed to BETA.  I will raise a new pull request once this is merged to improve this